index.md

```diff
@@ -5,7 +5,12 @@ title: Mod 0 Curriculum
 
 ## Tech Setup
 
-Complete the following steps before Mod 0 in order to get the tools you'll need on your machine. 
+Complete the following steps before Mod 0 in order to get the tools you'll need on your machine.
+
+If you'd like to watch a video overview, a Turing alum who is <strike>at least partially competent</strike> gainfully employed in the software development industry put this video together for you:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/7TTt1a8UHic" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Really enjoyed this. I thought it was paced well and included a lot of helpful tips in addition to walking folks through the installation. Also got a kick out some of the on screen comments.

Reply:
Love it embedded to. Videos are so helpful when starting out, and I like that it draws them there first.